Chartboost/ChartboostAdapterConfiguration.m

```diff
@@ -57,22 +51,23 @@ - (NSString *)networkSdkVersion {
 }
 
 - (void)initializeNetworkWithConfiguration:(NSDictionary<NSString *, id> *)configuration
-                                  complete:(void(^)(NSError *))complete {
-    
-    NSString * appId = configuration[kChartboostAppIdKey];
-    if (appId == nil) {
-        NSError * error = [NSError errorWithDomain:kAdapterErrorDomain code:ChartboostAdapterErrorCodeMissingAppId userInfo:@{ NSLocalizedDescriptionKey: @"Chartboost's initialization skipped. The appId is empty. Ensure it is properly configured on the MoPub dashboard. Note that initialization on the first app launch is a no-op." }];
+                                  complete:(void(^)(NSError *))complete
+{
+    if (configuration[kChartboostAppIdKey] == nil) {
+        NSError *error = [NSError errorWithDomain:kAdapterErrorDomain
+                                             code:ChartboostAdapterErrorCodeMissingAppId
+                                         userInfo:@{ NSLocalizedDescriptionKey: @"Chartboost's initialization skipped. The appId is empty. Ensure it is properly configured on the MoPub dashboard." }];
```

Comment:
Good catch! For some reason I thought it was not happening any more, but it could happen. I recovered the message and added a comment with link to more detailed info